src/controllers/transfers.js

```diff
@@ -413,12 +413,44 @@ function * putRejection () {
   this.status = result.existed ? 200 : 201
 }
 
+/**
+ * @api {post} /messages Send Message
+ * @apiName SendMessage
+ * @apiGroup Transfer Methods
+ * @apiVersion 1.0.0
+ *
+ * @apiDescription Send a message to another account
```

Comment:
Might be worth noting here whether this is a reliable message delivery mechanism or not